test/unittest/source/test_service.cpp

```diff
@@ -248,6 +250,41 @@ namespace casual
 
       }
 
+      TEST( test_service, service_call_to_route__expect_metric_to_use_route_name)
+      {
+         common::unittest::Trace trace;
+
+         auto domain = casual::domain::unittest::manager( local::configuration::base, R"(
+domain:
+   servers:
+      -  path: ${CASUAL_MAKE_SOURCE_ROOT}/middleware/example/server/bin/casual-example-server
+         memberships: [ user]
+
+   services:
+      -  name: casual/example/echo
+         routes: [ a, b]
+)");
+
+         // Subscribe to the metric event
+         common::message::event::service::Calls event;
+         common::event::subscribe( common::process::handle(), { event.type()});
+
+         // Call service to generate metric
+         auto buffer = tpalloc( X_OCTET, nullptr, 128);
+         auto len = tptypes( buffer, nullptr, nullptr);
```

Comment:
A scope around the variables (and usage of the variables). Mitigates unwanted usage and a good practice in general to keep scopes as small as possible.